frontend/src/components/Landlord/Header.tsx

```diff
@@ -114,50 +118,64 @@ const LandlordHeader = ({ name, overallRating, numReviews, handleClick }: Props)
-                <Grid container direction="row" className={styles.HeaderRow} alignItems="flex-end">
-                  <Grid item xs={12} md={mobileView ? 12 : 3}>
-                    <Avatar
-                      src={Logo}
-                      alt="Landlord Logo"
-                      className={logo}
-                      style={
-                        mobileView
-                          ? { marginLeft: '5px', marginBottom: '10px' }
-                          : { marginLeft: '5%', marginBottom: '20px' }
-                      }
-                    />
-                  </Grid>
-                  <Grid
-                    item
-                    xs={12}
-                    md={mobileView ? 12 : 8}
-                    style={mobileView ? { marginLeft: '10px' } : {}}
-                  >
-                    <CardHeader title={name} className={landlordName} disableTypography={true} />
-                    <Grid container style={{ paddingTop: '10px' }}>
-                      <Grid
-                        item
-                        className={landlordRating}
-                        xs={12}
-                        md={mobileView ? 12 : 3}
-                        style={{ paddingTop: '5px', marginRight: '45px' }}
-                      >
-                        <HeartRating value={overallRating} readOnly />
-                      </Grid>
-                      <Grid item xs={12} md={mobileView ? 12 : 6}>
-                        <CardHeader
-                          title={numReviews + ' Reviews'}
-                          className={landlordReviews}
-                          disableTypography={true}
-                          style={mobileView ? { marginLeft: '0px' } : {}}
-                        />
+                <Grid
+                  container
+                  className={styles.HeaderRow}
+                  justify="space-between"
+                  alignItems="flex-end"
+                >
+                  <Grid container item xs={9} justify="flex-start">
+                    <Grid item md={mobileView ? 12 : 2}>
+                      <Avatar
+                        src={profilePhoto}
+                        alt={name}
+                        className={logo}
+                        style={
+                          mobileView
+                            ? { marginLeft: '5px', marginBottom: '10px' }
+                            : { marginLeft: '5%', marginBottom: '20px' }
+                        }
+                      />
+                    </Grid>
+                    <Grid item style={mobileView ? { marginLeft: '10px' } : {}}>
+                      <CardHeader title={name} className={landlordName} disableTypography={true} />
+                      <Grid container style={{ paddingTop: '10px' }}>
+                        <Grid
+                          item
+                          className={landlordRating}
+                          xs={12}
+                          md={mobileView ? 12 : 4}
```

Comment:
Not sure why you need to make this based on mobile view, unless it's glitching when you playtest it on a mobile screen size. The `md` size should account for non-mobile sizes anyway so you can leave it at 4 for screens medium and up.